STIPS/STIP-009.md

```diff
@@ -0,0 +1,1195 @@
+# STIP-009: Self Service Manager Contracts
+
+*Using template v0.1*
+
+## Abstract
+
+Historically, SetProtocol has deployed SetToken specific manager contracts and extensions for IndexCoop issued funds. These contracts encode fee management and rebalance trading logic and provide better security guarantees to SetToken holders than direct management via EOA.
```

Comment:
nit: let's either use (SetProtocol and SetToken) or (Set Protocol and Set Token). we have both right now